boards/arm/frdm_k64f/frdm_k64f_bt.dts

```diff
@@ -0,0 +1,12 @@
+#include <frdm_k64f.dts>
```

Comment:
How do we select which of frdm_k64f or frdm_k64f_bt should be used in an application?

Reply:
same way we would any board.